onnxruntime/core/providers/dml/DmlExecutionProvider/src/GraphPartitioner.cpp

```diff
@@ -740,129 +659,30 @@
         return partitions;
     }
 
-    std::unordered_map<const onnx::TensorProto*, std::vector<uint32_t>>
-    GetInitializerToPartitionMap(
-        const onnxruntime::GraphViewer& graph,
-        gsl::span<std::unique_ptr<GraphPartition>> partitions
-    )
-    {
-        std::unordered_map<const onnx::TensorProto*, std::vector<uint32_t>> initializerPartitionMap;
-
-        for (uint32_t partitionIndex = 0; partitionIndex < gsl::narrow_cast<uint32_t>(partitions.size()); ++partitionIndex)
-        {
-            auto& partition = partitions[partitionIndex];
-
-            // Skip partitions which have been merged into other partitions
-            if (partition->GetRootMergedPartition() != partition.get())
-            {
-                continue;
-            }
-
-            std::unordered_map<std::string, onnx::TensorProto> transferredInitializerMap;
-
-            for (const std::string& input : partition->GetInputs())
-            {
-                const onnx::TensorProto* tensor = nullptr;
-                if (graph.GetInitializedTensor(input, tensor))
-                {
-                    initializerPartitionMap[tensor].push_back(partitionIndex);
-                }
-            }
-        }
-
-        return initializerPartitionMap;
-    }
-
     std::vector<std::unique_ptr<onnxruntime::ComputeCapability>>
-    PartitionGraph(
+    LightWeightPartitionGraph(
```

Comment:
>LightWeightPartitionGraph [](http://example.com/codeflow?start=4&length=25)

It would be cleaner to split this class now that fusion and GetCapability are separated.